### PR TITLE
chore: enable dns and discv5 discovery

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -71,6 +72,17 @@ services:
 {% if go_waku_rendezvous_server_enabled %}
       --rendezvous-server
       --rendezvous-db-path=/data/rendezvous
+{% endif %}
+{% if go_waku_dns_disc_enabled %}
+      --dns-discovery=true
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2]: I think I'd rather have the toggle flag outside of the if with the boolean formatted using `to_json`. Same below.

Comment (jakubgs) [reply templates/docker-compose.yml.j2]: This way we can explicitly see that dns discovery is disabled when it is in the flags.

**Review by jakubgs — APPROVED** [review]

**Review by jm-clius — APPROVED** [review]
LGTM.

File: templates/docker-compose.yml.j2 [hunk 2]
```diff
@@ -71,6 +72,17 @@ services:
 {% if go_waku_rendezvous_server_enabled %}
       --rendezvous-server
       --rendezvous-db-path=/data/rendezvous
+{% endif %}
+      --dns-discovery={{ go_waku_dns_disc_url | bool | to_json }}
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2:76]: Whoops, should not have merged this, you used the wrong variable here.

**Pull request merged**